core/templates/dev/head/pages/exploration_editor/publish_exploration_modal.html

```diff
@@ -0,0 +1,28 @@
+<script type="text/ng-template" id="modals/publishExploration">
```

Comment:
Has the existing code for this modal been deleted? Alternatively, you could get rid of this file and deal with it in a separate PR if it's related to @Keksike's changes.